.templates/docs.sh.template

```diff
@@ -42,17 +42,17 @@ EOF
     conda activate travis-ci-"$TRAVIS_JOB_NUMBER"
     cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
 
-    FPGA_CONFIG='$HOME/fpga_config'
+    FPGA_CONFIG="/home/fpga-ci/fpga_config"
```

Comment:
Was there a reason for this change?

Reply:
`$HOME` got expanded to `/home/travis` instead of the remote environemnt, I'm playing around with this now but even using single quotes seems to have not worked? not really sure why it's being cranky

Reply:
You need a backslash before the $ to have it not interpret the `$HOME` immediately and instead pass the string `$HOME` to the ssh command

Reply:
This never came up when I was googling around using local vs remote variables over ssh... not sure why, but thank you! :pray:

Reply:
It's not actually an SSH thing, it's a general bash string interpretation thing, it comes up a lot with [heredoc](https://unix.stackexchange.com/questions/68419/how-to-print-in-here-document) strings though